api/src/apps/boat_tracking/forms.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.gis import forms, geos
+
+from apps.boat_tracking import constants
+from apps.boat_tracking.models import BoatTracking
+
+
+class CustomPointField(forms.PointField):
+    def to_python(self, value):
+        return geos.Point(
+            value['x'],
+            value['y'],
+            srid=4326
```

Comment:
What is `srid=4326`? I think `4326` should be moved to a constant with a clear name. And also, why is it the same for all points?